src/DependencyInjection/Providers/GoogleProviderConfigurator.php

```diff
@@ -19,9 +19,11 @@ public function buildConfiguration(NodeBuilder $node)
         // todo - add the comments as help text, and render in README
         $node
             ->scalarNode('access_type')
+                ->defaultValue('')
                 ->info('Optional value for sending access_type parameter. More detail: https://developers.google.com/identity/protocols/OpenIDConnect#authenticationuriparameters')
             ->end()
             ->scalarNode('hosted_domain')
+                ->defaultValue('')
```

Comment:
Not sure, but maybe we need `null` instead for `access_type` and `hosted_domain` options?